ui/shared/components/panel/variants/Frequencies.jsx

```diff
@@ -157,11 +157,15 @@ const sectionTitle = ({ fieldTitle, section }) => (
 const HOM_SECTION = 'Homoplasmy'
 const HET_SECTION = 'Heteroplasmy'
 
+const SV_CALLSET_POP = { field: 'sv_callset', fieldTitle: 'This Callset', acDisplay: 'AC', helpMessage: SV_CALLSET_CRITERIA_MESSAGE }
 const CALLSET_POP = { field: 'callset', fieldTitle: 'This Callset', acDisplay: 'AC' }
+const SEQR_POP = { ...CALLSET_POP, field: 'seqr', fieldTitle: 'seqr' }
```

Comment:
Should the value for the `fieldTitle` be `seqr Callset`?

Reply:
see above, but no